frontend/src/admin/components/sidebar/dropdown/dropdownMenu.scss

```diff
@@ -0,0 +1,43 @@
+@import "src/styles/main";
+
+.c-menu {
+  position: relative;
+
+  &__button{
+    &--active {
+      .c-dropdown {
+        opacity: 1;
+        visibility: visible;
+        position: absolute;
```

Comment:
Is it necessary to position this absolutely? I'd prefer if this was inline with the "Users" title. That way you can get the browser render system to have the components react to one another on resizing; and it is less prone to changes like position or padding changes of the parent.

Reply:
I use the position absolute to make the menu-item display over the userList content and not pushing the content down. Is there a better way to do this?

Reply:
You can disregard this comment. I misunderstood what `.c-dropdown` was describing.